core/src/test/java/kafka/test/ClusterConfig.java

```diff
@@ -153,15 +148,19 @@ public Map<Integer, Map<String, String>> perBrokerOverrideProperties() {
         return perBrokerOverrideProperties;
     }
 
-    public Map<String, String> nameTags() {
-        Map<String, String> tags = new LinkedHashMap<>(4);
-        name().ifPresent(name -> tags.put("Name", name));
-        tags.put("MetadataVersion", metadataVersion.toString());
-        tags.put("Security", securityProtocol.name());
-        listenerName().ifPresent(listener -> tags.put("Listener", listener));
+    public String[] tags() {
         return tags;
     }
 
+    public Map<String, String> nameTags() {
```

Comment:
Could you please add the `tags` to this method? Also, it would be great to rename it to "displayName", and the return type should be `String` instead of `Map<String, String>`

Reply:
since we can get the tags from the getter, why do we still need to add this into the displayTags?

Reply:
My point was `Map<String, String> nameTags` should be changed to `Set<String> nameTags` (or `displayTags`). Also, it should include the "tags". For example:

```java
    public Set<String> nameTags() {
        Set<String> nameTags = new LinkedHashSet<>(tags);
        nameTags.add("MetadataVersion=" + metadataVersion.toString());
        nameTags.add("Security=" + securityProtocol.name());
        listenerName().ifPresent(listener -> nameTags.add("Listener=" + listener));
        return nameTags;
    }
```

Reply:
I see, basically also change the current nameTags to a List instead of a map. 
sure, let me address this, thanks!